ts/Dashboards/Components/Component.ts

```diff
@@ -1228,6 +1244,7 @@ namespace Component {
         caption?: TextOptionsType;
     }
 
+    /** @internal */
```

Comment:
I think this one should be external, otherwise, I wouldn't know what options I have:

![image](https://user-images.githubusercontent.com/1453926/227249688-fb71a3ed-b118-4061-9b90-10ca27f143cb.png)